internal/validator/service.go

```diff
@@ -0,0 +1,22 @@
+package validator
+
+import (
+	"context"
+
+	"github.com/eigerco/strawberry/internal/crypto"
+)
+
+type ValidatorService interface {
```

Comment:
This interface is essentially only used for mocks? (just checking)

Reply:
This is for the eventual logic for assurers, but it helps to have an interface for testing yes